docs/releases/2.2.txt

```diff
@@ -165,7 +165,10 @@ Forms
 Generic Views
 ~~~~~~~~~~~~~
 
-* ...
+* A new :meth:`~django.views.generic.base.View.init` hook initializes
```

Comment:
I think "The new" is more inline with other notes.

`View.init <django.views.generic.base.View.init>` would render "View" also which would help.